src/lib/products.ts

```diff
@@ -17,31 +17,27 @@ const productSlugsToNames: { [slug in ProductSlug]: ProductName } = {
 }
 
 /**
- * Holds the product slugs in the order ProductSwitcher will render them.
+ * An array of all Product slugs.
  */
-const productSwitcherSlugs: ProductSlug[][] = [
-  ['terraform', 'packer', 'vagrant'],
-  ['vault', 'boundary'],
-  ['consul'],
-  ['nomad', 'waypoint'],
-  ['hcp'],
-  ['sentinel'],
+const productSlugs: ProductSlug[] = [
```

Comment:
Can we generate the slugs from the productSlugsToNames object vs needing to manage two separate instances of slugs?

```
const productSlugs = Object.keys(productSlugsToNames)
```